downloadDirectory/src/main/resources/config.properties

```diff
@@ -9,3 +9,11 @@ absoluteReleaseDirectoryPath=/usr/local/gkb/scripts/release/
 releaseDownloadDirectoryPath=/usr/local/gkb/scripts/release/download_directory/
```

Comment:
These two default paths need to be updated -- though they might already be in an unmerged branch